[2025-04-29 19:20:46,784] INFO in _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
[2025-04-29 19:20:46,784] INFO in _internal: [33mPress CTRL+C to quit[0m
[2025-04-29 19:20:46,816] INFO in _internal:  * Restarting with watchdog (windowsapi)
[2025-04-29 19:20:47,092] WARNING in _internal:  * Debugger is active!
[2025-04-29 19:20:47,092] INFO in _internal:  * Debugger PIN: 131-854-612
[2025-04-29 19:20:54,111] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\IPython\\__init__.py', reloading
[2025-04-29 19:20:54,116] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\IPython\\terminal\\embed.py', reloading
[2025-04-29 19:20:54,159] INFO in _internal:  * Restarting with watchdog (windowsapi)
[2025-04-29 19:20:54,416] WARNING in _internal:  * Debugger is active!
[2025-04-29 19:20:54,416] INFO in _internal:  * Debugger PIN: 131-854-612
[2025-04-29 19:20:54,432] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\matplotlib\\style\\__init__.py', reloading
[2025-04-29 19:20:54,467] INFO in _internal:  * Restarting with watchdog (windowsapi)
[2025-04-29 19:20:54,690] WARNING in _internal:  * Debugger is active!
[2025-04-29 19:20:54,696] INFO in _internal:  * Debugger PIN: 131-854-612
[2025-04-29 19:20:54,742] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\optimize\\__init__.py', reloading
[2025-04-29 19:20:54,749] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\optimize\\_optimize.py', reloading
[2025-04-29 19:20:54,770] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\optimize\\_minimize.py', reloading
[2025-04-29 19:20:54,775] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\optimize\\_root.py', reloading
[2025-04-29 19:20:54,779] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\optimize\\_root_scalar.py', reloading
[2025-04-29 19:20:54,785] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\optimize\\_minpack_py.py', reloading
[2025-04-29 19:20:54,793] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\optimize\\_zeros_py.py', reloading
[2025-04-29 19:20:54,800] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\optimize\\_nonlin.py', reloading
[2025-04-29 19:20:54,802] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\special\\__init__.py', reloading
[2025-04-29 19:20:54,825] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\special\\_basic.py', reloading
[2025-04-29 19:20:54,833] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\special\\_support_alternative_backends.py', reloading
[2025-04-29 19:20:54,890] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\_pytest\\doctest.py', reloading
[2025-04-29 19:20:54,901] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\threadpoolctl.py', reloading
[2025-04-29 19:20:54,997] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\__init__.py', reloading
[2025-04-29 19:20:55,012] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\stats\\__init__.py', reloading
[2025-04-29 19:20:55,024] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\stats\\_stats_py.py', reloading
[2025-04-29 19:20:55,041] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\stats\\distributions.py', reloading
[2025-04-29 19:20:55,041] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\stats\\_continuous_distns.py', reloading
[2025-04-29 19:20:55,074] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\stats\\_discrete_distns.py', reloading
[2025-04-29 19:20:55,089] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\stats\\_morestats.py', reloading
[2025-04-29 19:20:55,099] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\stats\\_multicomp.py', reloading
[2025-04-29 19:20:55,106] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\stats\\_binned_statistic.py', reloading
[2025-04-29 19:20:55,116] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\stats\\_multivariate.py', reloading
[2025-04-29 19:20:55,128] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\stats\\_entropy.py', reloading
[2025-04-29 19:20:55,132] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\stats\\_hypotests.py', reloading
[2025-04-29 19:20:55,135] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\stats\\_sensitivity_analysis.py', reloading
[2025-04-29 19:20:55,141] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\stats\\_survival.py', reloading
[2025-04-29 19:20:55,149] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\fftpack\\__init__.py', reloading
[2025-04-29 19:20:55,152] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\fftpack\\_basic.py', reloading
[2025-04-29 19:20:55,152] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\fftpack\\_pseudo_diffs.py', reloading
[2025-04-29 19:20:55,156] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\fftpack\\_helper.py', reloading
[2025-04-29 19:20:55,164] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\fftpack\\_realtransforms.py', reloading
[2025-04-29 19:20:55,169] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\linalg\\__init__.py', reloading
[2025-04-29 19:20:55,174] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\linalg\\_misc.py', reloading
[2025-04-29 19:20:55,182] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\linalg\\_basic.py', reloading
[2025-04-29 19:20:55,185] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\linalg\\_decomp.py', reloading
[2025-04-29 19:20:55,194] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\linalg\\_decomp_lu.py', reloading
[2025-04-29 19:20:55,199] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\linalg\\_decomp_ldl.py', reloading
[2025-04-29 19:20:55,202] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\linalg\\_decomp_cholesky.py', reloading
[2025-04-29 19:20:55,207] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\linalg\\_decomp_qr.py', reloading
[2025-04-29 19:20:55,207] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\linalg\\_decomp_qz.py', reloading
[2025-04-29 19:20:55,211] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\linalg\\_decomp_svd.py', reloading
[2025-04-29 19:20:55,215] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\linalg\\_decomp_schur.py', reloading
[2025-04-29 19:20:55,219] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\linalg\\_decomp_polar.py', reloading
[2025-04-29 19:20:55,219] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\linalg\\_matfuncs.py', reloading
[2025-04-29 19:20:55,223] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\linalg\\blas.py', reloading
[2025-04-29 19:20:55,227] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\linalg\\lapack.py', reloading
[2025-04-29 19:20:55,232] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\linalg\\_special_matrices.py', reloading
[2025-04-29 19:20:55,235] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\linalg\\_solvers.py', reloading
[2025-04-29 19:20:55,239] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\linalg\\_procrustes.py', reloading
[2025-04-29 19:20:55,239] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\linalg\\_sketches.py', reloading
[2025-04-29 19:20:55,243] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\linalg\\_decomp_cossin.py', reloading
[2025-04-29 19:20:55,306] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\special\\_logsumexp.py', reloading
[2025-04-29 19:20:55,346] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\sparse\\__init__.py', reloading
[2025-04-29 19:20:55,354] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\sparse\\_base.py', reloading
[2025-04-29 19:20:55,360] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\sparse\\_csr.py', reloading
[2025-04-29 19:20:55,366] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\sparse\\_csc.py', reloading
[2025-04-29 19:20:55,368] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\sparse\\_lil.py', reloading
[2025-04-29 19:20:55,370] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\sparse\\_dok.py', reloading
[2025-04-29 19:20:55,378] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\sparse\\_coo.py', reloading
[2025-04-29 19:20:55,381] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\sparse\\_dia.py', reloading
[2025-04-29 19:20:55,386] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\sparse\\_bsr.py', reloading
[2025-04-29 19:20:55,387] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\sparse\\_construct.py', reloading
[2025-04-29 19:20:55,394] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\sparse\\_extract.py', reloading
[2025-04-29 19:20:55,395] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\sparse\\_matrix_io.py', reloading
[2025-04-29 19:20:55,403] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\concurrent\\__init__.py', reloading
[2025-04-29 19:20:55,403] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\idlelib\\__init__.py', reloading
[2025-04-29 19:20:55,403] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\lib2to3\\__init__.py', reloading
[2025-04-29 19:20:55,409] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\msilib\\__init__.py', reloading
[2025-04-29 19:20:55,409] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\test\\__init__.py', reloading
[2025-04-29 19:20:55,412] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\turtledemo\\__init__.py', reloading
[2025-04-29 19:20:55,413] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\wsgiref\\__init__.py', reloading
[2025-04-29 19:20:55,413] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\xmlrpc\\__init__.py', reloading
[2025-04-29 19:20:55,413] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\__phello__\\__init__.py', reloading
[2025-04-29 19:20:55,417] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\adodbapi\\__init__.py', reloading
[2025-04-29 19:20:55,420] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\aiobotocore\\__init__.py', reloading
[2025-04-29 19:20:55,420] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\aiohappyeyeballs\\__init__.py', reloading
[2025-04-29 19:20:55,436] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\aiohappyeyeballs\\impl.py', reloading
[2025-04-29 19:20:55,446] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\aiohappyeyeballs\\types.py', reloading
[2025-04-29 19:20:55,450] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\aiohappyeyeballs\\utils.py', reloading
[2025-04-29 19:20:55,453] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\aiohttp\\__init__.py', reloading
[2025-04-29 19:20:55,458] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\aiohttp\\client.py', reloading
[2025-04-29 19:20:55,476] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\aiohttp\\connector.py', reloading
[2025-04-29 19:20:55,484] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\aiohttp\\client_exceptions.py', reloading
[2025-04-29 19:20:55,490] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\aiohttp\\client_reqrep.py', reloading
[2025-04-29 19:20:55,493] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\aiohttp\\client_ws.py', reloading
[2025-04-29 19:20:55,500] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\aiohttp\\cookiejar.py', reloading
[2025-04-29 19:20:55,507] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\aiohttp\\formdata.py', reloading
[2025-04-29 19:20:55,510] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\aiohttp\\helpers.py', reloading
[2025-04-29 19:20:55,517] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\aiohttp\\http.py', reloading
[2025-04-29 19:20:55,523] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\aiohttp\\http_writer.py', reloading
[2025-04-29 19:20:55,525] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\aiohttp\\http_websocket.py', reloading
[2025-04-29 19:20:55,534] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\aiohttp\\multipart.py', reloading
[2025-04-29 19:20:55,539] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\aiohttp\\payload.py', reloading
[2025-04-29 19:20:55,544] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\aiohttp\\payload_streamer.py', reloading
[2025-04-29 19:20:55,544] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\aiohttp\\resolver.py', reloading
[2025-04-29 19:20:55,550] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\aiohttp\\streams.py', reloading
[2025-04-29 19:20:55,551] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\aiohttp\\tracing.py', reloading
[2025-04-29 19:20:55,555] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\aiohttp\\worker.py', reloading
[2025-04-29 19:20:55,563] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\aioitertools\\__init__.py', reloading
[2025-04-29 19:20:55,567] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\alabaster\\__init__.py', reloading
[2025-04-29 19:20:55,573] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\altair\\__init__.py', reloading
[2025-04-29 19:20:55,575] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\altair\\vegalite\\__init__.py', reloading
[2025-04-29 19:20:55,579] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\altair\\vegalite\\v5\\__init__.py', reloading
[2025-04-29 19:20:55,585] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\altair\\vegalite\\v5\\schema\\__init__.py', reloading
[2025-04-29 19:20:55,592] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\altair\\vegalite\\v5\\schema\\core.py', reloading
[2025-04-29 19:20:55,616] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\altair\\vegalite\\v5\\schema\\channels.py', reloading
[2025-04-29 19:20:55,650] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\altair\\vegalite\\v5\\api.py', reloading
[2025-04-29 19:20:55,665] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\altair\\utils\\schemapi.py', reloading
[2025-04-29 19:20:55,674] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\altair\\utils\\__init__.py', reloading
[2025-04-29 19:20:55,683] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\altair\\utils\\core.py', reloading
[2025-04-29 19:20:55,690] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\altair\\vegalite\\v5\\data.py', reloading
[2025-04-29 19:20:55,692] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\altair\\vegalite\\v5\\display.py', reloading
[2025-04-29 19:20:55,695] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\altair\\vegalite\\v5\\theme.py', reloading
[2025-04-29 19:20:55,699] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\altair\\expr\\__init__.py', reloading
[2025-04-29 19:20:55,700] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\altair\\expr\\funcs.py', reloading
[2025-04-29 19:20:55,706] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\altair\\expr\\consts.py', reloading
[2025-04-29 19:20:55,707] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\altair\\expr\\core.py', reloading
[2025-04-29 19:20:55,707] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\altair\\vegalite\\data.py', reloading
[2025-04-29 19:20:55,742] INFO in _internal:  * Restarting with watchdog (windowsapi)
[2025-04-29 19:20:55,974] WARNING in _internal:  * Debugger is active!
[2025-04-29 19:20:55,974] INFO in _internal:  * Debugger PIN: 131-854-612
[2025-04-29 19:20:55,999] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\colorcet\\__init__.py', reloading
[2025-04-29 19:20:56,003] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\comm\\__init__.py', reloading
[2025-04-29 19:20:56,008] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\conda\\__init__.py', reloading
[2025-04-29 19:20:56,017] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\conda\\_version.py', reloading
[2025-04-29 19:20:56,017] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\conda_build\\__init__.py', reloading
[2025-04-29 19:20:56,024] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\conda_build\\_version.py', reloading
[2025-04-29 19:20:56,025] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\conda_content_trust\\__init__.py', reloading
[2025-04-29 19:20:56,025] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\conda_env\\__init__.py', reloading
[2025-04-29 19:20:56,025] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\conda_index\\__init__.py', reloading
[2025-04-29 19:20:56,030] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\conda_libmamba_solver\\__init__.py', reloading
[2025-04-29 19:20:56,033] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\conda_pack\\__init__.py', reloading
[2025-04-29 19:20:56,033] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\conda_package_handling\\__init__.py', reloading
[2025-04-29 19:20:56,036] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\conda_package_streaming\\__init__.py', reloading
[2025-04-29 19:20:56,036] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\conda_token\\__init__.py', reloading
[2025-04-29 19:20:56,039] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\constantly\\__init__.py', reloading
[2025-04-29 19:20:56,041] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\constantly\\_constants.py', reloading
[2025-04-29 19:20:56,045] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\contourpy\\__init__.py', reloading
[2025-04-29 19:20:56,054] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\contourpy\\_version.py', reloading
[2025-04-29 19:20:56,056] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\contourpy\\convert.py', reloading
[2025-04-29 19:20:56,059] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\contourpy\\dechunk.py', reloading
[2025-04-29 19:20:56,063] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\cookiecutter\\__init__.py', reloading
[2025-04-29 19:20:56,066] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\cpuinfo\\__init__.py', reloading
[2025-04-29 19:20:56,066] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\cryptography\\__init__.py', reloading
[2025-04-29 19:20:56,066] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\cryptography\\__about__.py', reloading
[2025-04-29 19:20:56,075] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\cssselect\\__init__.py', reloading
[2025-04-29 19:20:56,080] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\cssselect\\parser.py', reloading
[2025-04-29 19:20:56,088] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\cssselect\\xpath.py', reloading
[2025-04-29 19:20:56,091] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\curl\\__init__.py', reloading
[2025-04-29 19:20:56,091] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\cytoolz\\__init__.py', reloading
[2025-04-29 19:20:56,091] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\dask\\__init__.py', reloading
[2025-04-29 19:20:56,096] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\dask_expr\\__init__.py', reloading
[2025-04-29 19:20:56,104] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\datashader\\__init__.py', reloading
[2025-04-29 19:20:56,105] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\debugpy\\__init__.py', reloading
[2025-04-29 19:20:56,112] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\debugpy\\public_api.py', reloading
[2025-04-29 19:20:56,116] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\diff_match_patch\\__init__.py', reloading
[2025-04-29 19:20:56,120] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\dill\\__init__.py', reloading
[2025-04-29 19:20:56,120] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\distributed\\__init__.py', reloading
[2025-04-29 19:20:56,133] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\distributed\\_version.py', reloading
[2025-04-29 19:20:56,136] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\distributed\\actor.py', reloading
[2025-04-29 19:20:56,141] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\distributed\\client.py', reloading
[2025-04-29 19:20:56,168] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\distributed\\core.py', reloading
[2025-04-29 19:20:56,175] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\distributed\\comm\\__init__.py', reloading
[2025-04-29 19:20:56,175] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\distributed\\comm\\core.py', reloading
[2025-04-29 19:20:56,182] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\distributed\\deploy\\__init__.py', reloading
[2025-04-29 19:20:56,184] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\distributed\\deploy\\adaptive.py', reloading
[2025-04-29 19:20:56,184] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\distributed\\deploy\\local.py', reloading
[2025-04-29 19:20:56,193] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\distributed\\deploy\\spec.py', reloading
[2025-04-29 19:20:56,195] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\distributed\\deploy\\ssh.py', reloading
[2025-04-29 19:20:56,200] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\distributed\\deploy\\subprocess.py', reloading
[2025-04-29 19:20:56,202] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\distributed\\diagnostics\\plugin.py', reloading
[2025-04-29 19:20:56,208] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\distributed\\diagnostics\\progressbar.py', reloading
[2025-04-29 19:20:56,215] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\distributed\\event.py', reloading
[2025-04-29 19:20:56,216] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\distributed\\lock.py', reloading
[2025-04-29 19:20:56,223] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\distributed\\multi_lock.py', reloading
[2025-04-29 19:20:56,225] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\distributed\\nanny.py', reloading
[2025-04-29 19:20:56,232] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\distributed\\pubsub.py', reloading
[2025-04-29 19:20:56,236] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\distributed\\queues.py', reloading
[2025-04-29 19:20:56,241] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\distributed\\scheduler.py', reloading
[2025-04-29 19:20:56,268] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\distributed\\security.py', reloading
[2025-04-29 19:20:56,268] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\distributed\\semaphore.py', reloading
[2025-04-29 19:20:56,275] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\distributed\\spans.py', reloading
[2025-04-29 19:20:56,275] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\distributed\\threadpoolexecutor.py', reloading
[2025-04-29 19:20:56,275] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\distributed\\utils.py', reloading
[2025-04-29 19:20:56,315] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\distributed\\variable.py', reloading
[2025-04-29 19:20:56,315] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\distributed\\worker.py', reloading
[2025-04-29 19:20:56,326] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\distributed\\exceptions.py', reloading
[2025-04-29 19:20:56,332] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\distributed\\worker_client.py', reloading
[2025-04-29 19:20:56,334] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\distro\\__init__.py', reloading
[2025-04-29 19:20:56,341] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\distro\\distro.py', reloading
[2025-04-29 19:20:56,341] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\docstring_to_markdown\\__init__.py', reloading
[2025-04-29 19:20:56,352] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\dotenv\\__init__.py', reloading
[2025-04-29 19:20:56,352] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\dotenv\\main.py', reloading
[2025-04-29 19:20:56,359] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\erfa\\__init__.py', reloading
[2025-04-29 19:20:56,362] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\et_xmlfile\\__init__.py', reloading
[2025-04-29 19:20:56,362] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\executing\\__init__.py', reloading
[2025-04-29 19:20:56,367] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\executing\\executing.py', reloading
[2025-04-29 19:20:56,368] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\fastjsonschema\\__init__.py', reloading
[2025-04-29 19:20:56,375] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\fastjsonschema\\exceptions.py', reloading
[2025-04-29 19:20:56,375] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\fastjsonschema\\version.py', reloading
[2025-04-29 19:20:56,379] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\filelock\\__init__.py', reloading
[2025-04-29 19:20:56,382] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\filelock\\_api.py', reloading
[2025-04-29 19:20:56,383] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\filelock\\_error.py', reloading
[2025-04-29 19:20:56,387] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\filelock\\_soft.py', reloading
[2025-04-29 19:20:56,390] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\filelock\\_unix.py', reloading
[2025-04-29 19:20:56,392] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\filelock\\_windows.py', reloading
[2025-04-29 19:20:56,400] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\flask\\blueprints.py', reloading
[2025-04-29 19:20:56,406] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\flask\\sansio\\blueprints.py', reloading
[2025-04-29 19:20:56,410] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\flask\\cli.py', reloading
[2025-04-29 19:20:56,416] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\flask\\ctx.py', reloading
[2025-04-29 19:20:56,423] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\flask\\debughelpers.py', reloading
[2025-04-29 19:20:56,425] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\flask\\helpers.py', reloading
[2025-04-29 19:20:56,432] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\flask\\json\\__init__.py', reloading
[2025-04-29 19:20:56,434] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\flask\\logging.py', reloading
[2025-04-29 19:20:56,434] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\flask\\sessions.py', reloading
[2025-04-29 19:20:56,444] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\flask\\signals.py', reloading
[2025-04-29 19:20:56,451] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\flask\\testing.py', reloading
[2025-04-29 19:20:56,457] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\flask\\views.py', reloading
[2025-04-29 19:20:56,459] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\flask\\__main__.py', reloading
[2025-04-29 19:20:56,459] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\fontTools\\__init__.py', reloading
[2025-04-29 19:20:56,467] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\fontTools\\misc\\loggingTools.py', reloading
[2025-04-29 19:20:56,476] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\fsspec\\__init__.py', reloading
[2025-04-29 19:20:56,476] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\fsspec\\callbacks.py', reloading
[2025-04-29 19:20:56,485] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\fsspec\\compression.py', reloading
[2025-04-29 19:20:56,492] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\fsspec\\core.py', reloading
[2025-04-29 19:20:56,499] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\fsspec\\exceptions.py', reloading
[2025-04-29 19:20:56,500] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\fsspec\\mapping.py', reloading
[2025-04-29 19:20:56,504] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\fsspec\\registry.py', reloading
[2025-04-29 19:20:56,509] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\fsspec\\spec.py', reloading
[2025-04-29 19:20:56,515] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\gensim\\__init__.py', reloading
[2025-04-29 19:20:56,517] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\git\\__init__.py', reloading
[2025-04-29 19:20:56,525] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\gitdb\\util.py', reloading
[2025-04-29 19:20:56,531] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\git\\exc.py', reloading
[2025-04-29 19:20:56,533] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\gitdb\\exc.py', reloading
[2025-04-29 19:20:56,533] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\git\\types.py', reloading
[2025-04-29 19:20:56,542] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\git\\compat.py', reloading
[2025-04-29 19:20:56,542] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\git\\config.py', reloading
[2025-04-29 19:20:56,551] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\git\\objects\\__init__.py', reloading
[2025-04-29 19:20:56,556] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\git\\objects\\blob.py', reloading
[2025-04-29 19:20:56,558] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\git\\objects\\commit.py', reloading
[2025-04-29 19:20:56,564] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\git\\objects\\base.py', reloading
[2025-04-29 19:20:56,568] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\git\\objects\\submodule\\__init__.py', reloading
[2025-04-29 19:20:56,571] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\git\\objects\\submodule\\root.py', reloading
[2025-04-29 19:20:56,573] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\git\\objects\\submodule\\base.py', reloading
[2025-04-29 19:20:56,575] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\git\\objects\\tag.py', reloading
[2025-04-29 19:20:56,586] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\git\\objects\\tree.py', reloading
[2025-04-29 19:20:56,599] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\git\\refs\\__init__.py', reloading
[2025-04-29 19:20:56,601] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\git\\refs\\head.py', reloading
[2025-04-29 19:20:56,601] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\git\\refs\\log.py', reloading
[2025-04-29 19:20:56,610] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\git\\refs\\reference.py', reloading
[2025-04-29 19:20:56,615] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\git\\refs\\remote.py', reloading
[2025-04-29 19:20:56,619] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\git\\refs\\symbolic.py', reloading
[2025-04-29 19:20:56,625] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\git\\refs\\tag.py', reloading
[2025-04-29 19:20:56,634] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\git\\diff.py', reloading
[2025-04-29 19:20:56,642] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\git\\db.py', reloading
[2025-04-29 19:20:56,642] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\gitdb\\db\\__init__.py', reloading
[2025-04-29 19:20:56,652] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\gitdb\\db\\base.py', reloading
[2025-04-29 19:20:56,658] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\gitdb\\db\\loose.py', reloading
[2025-04-29 19:20:56,659] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\gitdb\\db\\mem.py', reloading
[2025-04-29 19:20:56,665] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\gitdb\\db\\pack.py', reloading
[2025-04-29 19:20:56,673] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\gitdb\\db\\git.py', reloading
[2025-04-29 19:20:56,675] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\gitdb\\db\\ref.py', reloading
[2025-04-29 19:20:56,682] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\git\\cmd.py', reloading
[2025-04-29 19:20:56,690] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\git\\repo\\__init__.py', reloading
[2025-04-29 19:20:56,692] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\git\\repo\\base.py', reloading
[2025-04-29 19:20:56,700] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\git\\remote.py', reloading
[2025-04-29 19:20:56,709] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\git\\util.py', reloading
[2025-04-29 19:20:56,709] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\git\\index\\__init__.py', reloading
[2025-04-29 19:20:56,717] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\git\\index\\typ.py', reloading
[2025-04-29 19:20:56,720] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\git\\index\\base.py', reloading
[2025-04-29 19:20:56,724] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\gitdb\\__init__.py', reloading
[2025-04-29 19:20:56,725] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\h11\\__init__.py', reloading
[2025-04-29 19:20:56,725] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\h11\\_connection.py', reloading
[2025-04-29 19:20:56,735] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\h11\\_events.py', reloading
[2025-04-29 19:20:56,741] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\h11\\_state.py', reloading
[2025-04-29 19:20:56,748] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\h11\\_util.py', reloading
[2025-04-29 19:20:56,750] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\h5py\\__init__.py', reloading
[2025-04-29 19:20:56,750] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\holoviews\\__init__.py', reloading
[2025-04-29 19:20:56,758] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\holoviews\\element\\__init__.py', reloading
[2025-04-29 19:20:56,765] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\holoviews\\element\\annotation.py', reloading
[2025-04-29 19:20:56,767] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\holoviews\\element\\chart.py', reloading
[2025-04-29 19:20:56,771] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\holoviews\\element\\chart3d.py', reloading
[2025-04-29 19:20:56,773] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\holoviews\\element\\geom.py', reloading
[2025-04-29 19:20:56,775] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\holoviews\\element\\graphs.py', reloading
[2025-04-29 19:20:56,783] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\holoviews\\element\\path.py', reloading
[2025-04-29 19:20:56,786] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\holoviews\\element\\raster.py', reloading
[2025-04-29 19:20:56,791] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\holoviews\\element\\sankey.py', reloading
[2025-04-29 19:20:56,792] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\holoviews\\element\\stats.py', reloading
[2025-04-29 19:20:56,792] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\holoviews\\element\\tabular.py', reloading
[2025-04-29 19:20:56,799] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\holoviews\\element\\tiles.py', reloading
[2025-04-29 19:20:56,803] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\holoviews\\__version.py', reloading
[2025-04-29 19:20:56,806] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\holoviews\\annotators.py', reloading
[2025-04-29 19:20:56,815] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\httpcore\\__init__.py', reloading
[2025-04-29 19:20:56,817] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\httpcore\\_api.py', reloading
[2025-04-29 19:20:56,817] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\httpcore\\_async\\__init__.py', reloading
[2025-04-29 19:20:56,825] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\httpcore\\_async\\interfaces.py', reloading
[2025-04-29 19:20:56,825] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\httpcore\\_async\\connection_pool.py', reloading
[2025-04-29 19:20:56,825] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\httpcore\\_async\\http2.py', reloading
[2025-04-29 19:20:56,835] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\httpcore\\_async\\http11.py', reloading
[2025-04-29 19:20:56,838] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\httpcore\\_async\\connection.py', reloading
[2025-04-29 19:20:56,842] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\httpcore\\_async\\http_proxy.py', reloading
[2025-04-29 19:20:56,842] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\httpcore\\_async\\socks_proxy.py', reloading
[2025-04-29 19:20:56,848] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\httpcore\\_backends\\base.py', reloading
[2025-04-29 19:20:56,850] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\httpcore\\_backends\\mock.py', reloading
[2025-04-29 19:20:56,850] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\httpcore\\_backends\\sync.py', reloading
[2025-04-29 19:20:56,856] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\httpcore\\_exceptions.py', reloading
[2025-04-29 19:20:56,858] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\httpcore\\_models.py', reloading
[2025-04-29 19:20:56,858] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\httpcore\\_ssl.py', reloading
[2025-04-29 19:20:56,863] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\httpcore\\_sync\\__init__.py', reloading
[2025-04-29 19:20:56,867] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\httpcore\\_sync\\interfaces.py', reloading
[2025-04-29 19:20:56,867] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\httpcore\\_sync\\connection_pool.py', reloading
[2025-04-29 19:20:56,871] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\httpcore\\_sync\\http2.py', reloading
[2025-04-29 19:20:56,875] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\httpcore\\_sync\\http11.py', reloading
[2025-04-29 19:20:56,875] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\httpcore\\_sync\\connection.py', reloading
[2025-04-29 19:20:56,884] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\httpcore\\_sync\\http_proxy.py', reloading
[2025-04-29 19:20:56,884] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\httpcore\\_sync\\socks_proxy.py', reloading
[2025-04-29 19:20:56,890] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\httpx\\__init__.py', reloading
[2025-04-29 19:20:56,894] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\httpx\\__version__.py', reloading
[2025-04-29 19:20:56,897] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\httpx\\_api.py', reloading
[2025-04-29 19:20:56,900] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\httpx\\_auth.py', reloading
[2025-04-29 19:20:56,900] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\httpx\\_client.py', reloading
[2025-04-29 19:20:56,908] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\httpx\\_config.py', reloading
[2025-04-29 19:20:56,911] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\httpx\\_content.py', reloading
[2025-04-29 19:20:56,914] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\httpx\\_exceptions.py', reloading
[2025-04-29 19:20:56,916] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\httpx\\_models.py', reloading
[2025-04-29 19:20:56,921] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\httpx\\_status_codes.py', reloading
[2025-04-29 19:20:56,925] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\httpx\\_transports\\asgi.py', reloading
[2025-04-29 19:20:56,925] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\httpx\\_transports\\base.py', reloading
[2025-04-29 19:20:56,930] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\httpx\\_transports\\default.py', reloading
[2025-04-29 19:20:56,933] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\httpx\\_transports\\mock.py', reloading
[2025-04-29 19:20:56,937] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\httpx\\_transports\\wsgi.py', reloading
[2025-04-29 19:20:56,938] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\httpx\\_types.py', reloading
[2025-04-29 19:20:56,942] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\httpx\\_urls.py', reloading
[2025-04-29 19:20:56,948] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\hvplot\\__init__.py', reloading
[2025-04-29 19:20:56,950] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\hyperlink\\__init__.py', reloading
[2025-04-29 19:20:56,950] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\hyperlink\\_url.py', reloading
[2025-04-29 19:20:56,959] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\idna\\__init__.py', reloading
[2025-04-29 19:20:56,964] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\idna\\core.py', reloading
[2025-04-29 19:20:56,967] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\idna\\intranges.py', reloading
[2025-04-29 19:20:56,967] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\imagecodecs\\__init__.py', reloading
[2025-04-29 19:20:56,972] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\imagecodecs\\imagecodecs.py', reloading
[2025-04-29 19:20:56,975] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\imageio\\__init__.py', reloading
[2025-04-29 19:20:56,983] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\imagesize\\__init__.py', reloading
[2025-04-29 19:20:56,989] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\imagesize\\imagesize.py', reloading
[2025-04-29 19:20:57,017] INFO in _internal:  * Restarting with watchdog (windowsapi)
[2025-04-29 19:20:57,258] WARNING in _internal:  * Debugger is active!
[2025-04-29 19:20:57,258] INFO in _internal:  * Debugger PIN: 131-854-612
[2025-04-29 19:20:57,275] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\jupyter_server\\__init__.py', reloading
[2025-04-29 19:20:57,275] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\jupyter_server\\_version.py', reloading
[2025-04-29 19:20:57,275] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\jupyter_server\\base\\call_context.py', reloading
[2025-04-29 19:20:57,285] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\jupyter_server_terminals\\__init__.py', reloading
[2025-04-29 19:20:57,285] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\jwt\\__init__.py', reloading
[2025-04-29 19:20:57,290] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\jwt\\api_jwk.py', reloading
[2025-04-29 19:20:57,294] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\jwt\\api_jws.py', reloading
[2025-04-29 19:20:57,301] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\jwt\\api_jwt.py', reloading
[2025-04-29 19:20:57,301] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\jwt\\exceptions.py', reloading
[2025-04-29 19:20:57,307] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\jwt\\jwks_client.py', reloading
[2025-04-29 19:20:57,308] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\keyring\\__init__.py', reloading
[2025-04-29 19:20:57,308] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\keyring\\core.py', reloading
[2025-04-29 19:20:57,317] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\kiwisolver\\__init__.py', reloading
[2025-04-29 19:20:57,325] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\lazy_loader\\__init__.py', reloading
[2025-04-29 19:20:57,329] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\lazy_object_proxy\\__init__.py', reloading
[2025-04-29 19:20:57,333] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\lckr_jupyterlab_variableinspector\\__init__.py', reloading
[2025-04-29 19:20:57,333] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\libarchive\\__init__.py', reloading
[2025-04-29 19:20:57,342] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\linkify_it\\__init__.py', reloading
[2025-04-29 19:20:57,347] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\llvmlite\\__init__.py', reloading
[2025-04-29 19:20:57,347] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\lmdb\\__init__.py', reloading
[2025-04-29 19:20:57,350] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\lmdb\\cffi.py', reloading
[2025-04-29 19:20:57,359] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\locket\\__init__.py', reloading
[2025-04-29 19:20:57,367] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\lxml\\__init__.py', reloading
[2025-04-29 19:20:57,367] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\lz4\\__init__.py', reloading
[2025-04-29 19:20:57,367] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\markdown_it\\__init__.py', reloading
[2025-04-29 19:20:57,374] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\markdown_it\\main.py', reloading
[2025-04-29 19:20:57,376] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\markupsafe\\__init__.py', reloading
[2025-04-29 19:20:57,383] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\matplotlib\\pylab.py', reloading
[2025-04-29 19:20:58,308] INFO in _internal:  * Restarting with watchdog (windowsapi)
[2025-04-29 19:20:58,542] WARNING in _internal:  * Debugger is active!
[2025-04-29 19:20:58,542] INFO in _internal:  * Debugger PIN: 131-854-612
[2025-04-29 19:21:10,858] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\cycler.py', reloading
[2025-04-29 19:21:10,858] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\matplotlib\\backends\\registry.py', reloading
[2025-04-29 19:21:10,858] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\numpy\\typing\\__init__.py', reloading
[2025-04-29 19:21:10,872] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\numpy\\_typing\\__init__.py', reloading
[2025-04-29 19:21:10,875] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\numpy\\_typing\\_array_like.py', reloading
[2025-04-29 19:21:10,881] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\matplotlib\\typing.py', reloading
[2025-04-29 19:21:10,885] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\numpy\\_typing\\_extended_precision.py', reloading
[2025-04-29 19:21:10,998] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\matplotlib\\tests\\__init__.py', reloading
[2025-04-29 19:21:10,998] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\matplotlib_inline\\__init__.py', reloading
[2025-04-29 19:21:10,998] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\mdit_py_plugins\\__init__.py', reloading
[2025-04-29 19:21:10,998] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\mdurl\\__init__.py', reloading
[2025-04-29 19:21:11,023] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\mdurl\\_decode.py', reloading
[2025-04-29 19:21:11,023] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\mdurl\\_encode.py', reloading
[2025-04-29 19:21:11,023] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\mdurl\\_format.py', reloading
[2025-04-29 19:21:11,023] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\mdurl\\_parse.py', reloading
[2025-04-29 19:21:11,032] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\mdurl\\_url.py', reloading
[2025-04-29 19:21:11,032] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\menuinst\\__init__.py', reloading
[2025-04-29 19:21:11,039] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\menuinst\\api.py', reloading
[2025-04-29 19:21:11,041] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\menuinst\\_version.py', reloading
[2025-04-29 19:21:11,041] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\mistune\\__init__.py', reloading
[2025-04-29 19:21:11,049] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\mistune\\markdown.py', reloading
[2025-04-29 19:21:11,049] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\mistune\\block_parser.py', reloading
[2025-04-29 19:21:11,056] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\mistune\\inline_parser.py', reloading
[2025-04-29 19:21:11,056] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\mistune\\renderers.py', reloading
[2025-04-29 19:21:11,056] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\mistune\\util.py', reloading
[2025-04-29 19:21:11,056] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\mkl\\__init__.py', reloading
[2025-04-29 19:21:11,066] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\mkl_fft\\__init__.py', reloading
[2025-04-29 19:21:11,074] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\mkl_random\\__init__.py', reloading
[2025-04-29 19:21:11,088] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\mpmath\\__init__.py', reloading
[2025-04-29 19:21:11,089] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\msgpack\\__init__.py', reloading
[2025-04-29 19:21:11,089] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\multipledispatch\\__init__.py', reloading
[2025-04-29 19:21:11,089] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\mypy\\__init__.py', reloading
[2025-04-29 19:21:11,099] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\nacl\\__init__.py', reloading
[2025-04-29 19:21:11,099] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\navigator_updater\\__init__.py', reloading
[2025-04-29 19:21:11,099] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\nbclient\\__init__.py', reloading
[2025-04-29 19:21:11,105] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\nbconvert\\__init__.py', reloading
[2025-04-29 19:21:11,113] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\nbconvert\\_version.py', reloading
[2025-04-29 19:21:11,113] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\nbconvert\\exporters\\__init__.py', reloading
[2025-04-29 19:21:11,121] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\nbconvert\\exporters\\asciidoc.py', reloading
[2025-04-29 19:21:11,124] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\nbconvert\\exporters\\exporter.py', reloading
[2025-04-29 19:21:11,129] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\nbconvert\\exporters\\base.py', reloading
[2025-04-29 19:21:11,132] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\nbconvert\\exporters\\html.py', reloading
[2025-04-29 19:21:11,137] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\nbconvert\\exporters\\latex.py', reloading
[2025-04-29 19:21:11,142] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\nbconvert\\exporters\\markdown.py', reloading
[2025-04-29 19:21:11,145] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\nbconvert\\exporters\\notebook.py', reloading
[2025-04-29 19:21:11,145] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\nbconvert\\exporters\\pdf.py', reloading
[2025-04-29 19:21:11,150] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\nbconvert\\exporters\\python.py', reloading
[2025-04-29 19:21:11,152] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\nbconvert\\exporters\\qtpdf.py', reloading
[2025-04-29 19:21:11,154] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\nbconvert\\exporters\\qtpng.py', reloading
[2025-04-29 19:21:11,154] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\nbconvert\\exporters\\rst.py', reloading
[2025-04-29 19:21:11,157] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\nbconvert\\exporters\\script.py', reloading
[2025-04-29 19:21:11,160] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\nbconvert\\exporters\\slides.py', reloading
[2025-04-29 19:21:11,162] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\nbconvert\\exporters\\templateexporter.py', reloading
[2025-04-29 19:21:11,166] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\nbconvert\\exporters\\webpdf.py', reloading
[2025-04-29 19:21:11,170] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\nbformat\\__init__.py', reloading
[2025-04-29 19:21:11,174] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\nbformat\\_version.py', reloading
[2025-04-29 19:21:11,174] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\nbformat\\sentinel.py', reloading
[2025-04-29 19:21:11,179] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\nbformat\\converter.py', reloading
[2025-04-29 19:21:11,181] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\nbformat\\notebooknode.py', reloading
[2025-04-29 19:21:11,183] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\nbformat\\v4\\__init__.py', reloading
[2025-04-29 19:21:11,185] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\nbformat\\v4\\nbbase.py', reloading
[2025-04-29 19:21:11,186] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\nbformat\\validator.py', reloading
[2025-04-29 19:21:11,191] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\nbformat\\json_compat.py', reloading
[2025-04-29 19:21:11,194] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\nltk\\__init__.py', reloading
[2025-04-29 19:21:11,194] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\notebook\\__init__.py', reloading
[2025-04-29 19:21:11,199] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\notebook_shim\\__init__.py', reloading
[2025-04-29 19:21:11,202] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\numba\\__init__.py', reloading
[2025-04-29 19:21:11,224] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\numba\\core\\errors.py', reloading
[2025-04-29 19:21:11,225] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\numba\\core\\types\\__init__.py', reloading
[2025-04-29 19:21:11,233] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\numba\\core\\types\\abstract.py', reloading
[2025-04-29 19:21:11,233] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\numba\\core\\types\\containers.py', reloading
[2025-04-29 19:21:11,241] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\numba\\core\\types\\functions.py', reloading
[2025-04-29 19:21:11,241] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\numba\\core\\types\\iterators.py', reloading
[2025-04-29 19:21:11,246] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\numba\\core\\types\\misc.py', reloading
[2025-04-29 19:21:11,250] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\numba\\core\\types\\npytypes.py', reloading
[2025-04-29 19:21:11,257] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\numba\\core\\types\\scalars.py', reloading
[2025-04-29 19:21:11,258] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\numba\\core\\types\\function_type.py', reloading
[2025-04-29 19:21:11,264] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\numbergen\\__init__.py', reloading
[2025-04-29 19:21:11,272] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\numexpr\\__init__.py', reloading
[2025-04-29 19:21:11,282] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\numpydoc\\__init__.py', reloading
[2025-04-29 19:21:11,282] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\OpenSSL\\__init__.py', reloading
[2025-04-29 19:21:11,289] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\OpenSSL\\version.py', reloading
[2025-04-29 19:21:11,291] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\OpenSSL\\version.py', reloading
[2025-04-29 19:21:11,291] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\overrides\\__init__.py', reloading
[2025-04-29 19:21:11,297] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\overrides\\enforce.py', reloading
[2025-04-29 19:21:11,299] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\overrides\\overrides.py', reloading
[2025-04-29 19:21:11,299] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\packaging\\__init__.py', reloading
[2025-04-29 19:21:11,304] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\panel\\__init__.py', reloading
[2025-04-29 19:21:11,603] INFO in _internal:  * Restarting with watchdog (windowsapi)
[2025-04-29 19:21:11,825] WARNING in _internal:  * Debugger is active!
[2025-04-29 19:21:11,829] INFO in _internal:  * Debugger PIN: 131-854-612
[2025-04-29 19:21:34,801] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\param\\__init__.py', reloading
[2025-04-29 19:21:34,809] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\param\\reactive.py', reloading
[2025-04-29 19:21:34,809] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\panel\\config.py', reloading
[2025-04-29 19:21:34,822] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\panel\\__version.py', reloading
[2025-04-29 19:21:34,825] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\panel\\depends.py', reloading
[2025-04-29 19:21:34,827] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\panel\\io\\__init__.py', reloading
[2025-04-29 19:21:34,827] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\panel\\io\\cache.py', reloading
[2025-04-29 19:21:34,836] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\panel\\io\\notebook.py', reloading
[2025-04-29 19:21:34,851] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\panel\\io\\server.py', reloading
[2025-04-29 19:21:34,868] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\panel\\io\\state.py', reloading
[2025-04-29 19:21:34,876] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\panel\\layout\\__init__.py', reloading
[2025-04-29 19:21:34,878] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\panel\\layout\\accordion.py', reloading
[2025-04-29 19:21:34,880] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\panel\\layout\\card.py', reloading
[2025-04-29 19:21:34,915] INFO in _internal:  * Restarting with watchdog (windowsapi)
[2025-04-29 19:21:35,139] WARNING in _internal:  * Debugger is active!
[2025-04-29 19:21:35,139] INFO in _internal:  * Debugger PIN: 131-854-612
[2025-04-29 19:21:35,176] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\prompt_toolkit\\formatted_text\\__init__.py', reloading
[2025-04-29 19:21:35,178] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\prompt_toolkit\\formatted_text\\ansi.py', reloading
[2025-04-29 19:21:35,182] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\prompt_toolkit\\formatted_text\\html.py', reloading
[2025-04-29 19:21:35,185] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\prompt_toolkit\\shortcuts\\__init__.py', reloading
[2025-04-29 19:21:35,185] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\prompt_toolkit\\shortcuts\\prompt.py', reloading
[2025-04-29 19:21:35,200] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\prompt_toolkit\\shortcuts\\utils.py', reloading
[2025-04-29 19:21:35,203] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\ptyprocess\\__init__.py', reloading
[2025-04-29 19:21:35,207] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\pure_eval\\__init__.py', reloading
[2025-04-29 19:21:35,210] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\pyarrow\\__init__.py', reloading
[2025-04-29 19:21:35,210] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\pyasn1_modules\\__init__.py', reloading
[2025-04-29 19:21:35,215] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\pycparser\\__init__.py', reloading
[2025-04-29 19:21:35,217] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\pyct\\__init__.py', reloading
[2025-04-29 19:21:35,217] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\pydantic\\__init__.py', reloading
[2025-04-29 19:21:35,226] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\pydantic\\errors.py', reloading
[2025-04-29 19:21:35,233] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\pydantic\\main.py', reloading
[2025-04-29 19:21:35,240] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\pydantic\\networks.py', reloading
[2025-04-29 19:21:35,241] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\pydantic\\types.py', reloading
[2025-04-29 19:21:35,252] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\pydantic\\deprecated\\tools.py', reloading
[2025-04-29 19:21:35,252] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\pydantic\\version.py', reloading
[2025-04-29 19:21:35,258] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\pydantic_core\\core_schema.py', reloading
[2025-04-29 19:21:35,267] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\pydantic\\_internal\\_generate_schema.py', reloading
[2025-04-29 19:21:35,275] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\pydantic\\aliases.py', reloading
[2025-04-29 19:21:35,275] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\pydantic\\annotated_handlers.py', reloading
[2025-04-29 19:21:35,280] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\pydantic\\config.py', reloading
[2025-04-29 19:21:35,284] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\pydantic\\fields.py', reloading
[2025-04-29 19:21:35,284] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\pydantic\\functional_serializers.py', reloading
[2025-04-29 19:21:35,292] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\pydantic\\functional_validators.py', reloading
[2025-04-29 19:21:35,299] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\pydantic\\json_schema.py', reloading
[2025-04-29 19:21:35,307] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\pydantic\\type_adapter.py', reloading
[2025-04-29 19:21:35,308] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\pydantic\\validate_call_decorator.py', reloading
[2025-04-29 19:21:35,308] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\pydantic\\warnings.py', reloading
[2025-04-29 19:21:35,317] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\pydantic\\deprecated\\class_validators.py', reloading
[2025-04-29 19:21:35,317] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\pydantic\\deprecated\\config.py', reloading
[2025-04-29 19:21:35,322] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\pydantic\\root_model.py', reloading
[2025-04-29 19:21:35,325] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\pydantic_core\\__init__.py', reloading
[2025-04-29 19:21:35,332] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\pydeck\\__init__.py', reloading
[2025-04-29 19:21:35,334] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\pydispatch\\__init__.py', reloading
[2025-04-29 19:21:35,334] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\pydocstyle\\__init__.py', reloading
[2025-04-29 19:21:35,337] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\pylint\\__init__.py', reloading
[2025-04-29 19:21:35,340] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\pylint\\__pkginfo__.py', reloading
[2025-04-29 19:21:35,342] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\pylsp\\__init__.py', reloading
[2025-04-29 19:21:35,345] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\pylsp\\_version.py', reloading
[2025-04-29 19:21:35,350] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\pylsp_jsonrpc\\__init__.py', reloading
[2025-04-29 19:21:35,350] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\pylsp_jsonrpc\\_version.py', reloading
[2025-04-29 19:21:35,350] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\pyls_spyder\\__init__.py', reloading
[2025-04-29 19:21:35,356] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\pyparsing\\__init__.py', reloading
[2025-04-29 19:21:35,358] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\pyparsing\\util.py', reloading
[2025-04-29 19:21:35,358] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\pyparsing\\exceptions.py', reloading
[2025-04-29 19:21:35,364] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\pyparsing\\actions.py', reloading
[2025-04-29 19:21:35,366] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\pyparsing\\results.py', reloading
[2025-04-29 19:21:35,374] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\pyparsing\\core.py', reloading
[2025-04-29 19:21:35,385] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\pyparsing\\helpers.py', reloading
[2025-04-29 19:21:35,392] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\pyparsing\\unicode.py', reloading
[2025-04-29 19:21:35,432] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\PyQt5\\__init__.py', reloading
[2025-04-29 19:21:35,432] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\pytest\\__init__.py', reloading
[2025-04-29 19:21:35,443] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\_pytest\\__init__.py', reloading
[2025-04-29 19:21:35,443] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\_pytest\\_version.py', reloading
[2025-04-29 19:21:35,449] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\_pytest\\_code\\__init__.py', reloading
[2025-04-29 19:21:35,449] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\_pytest\\_code\\code.py', reloading
[2025-04-29 19:21:35,458] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\_pytest\\assertion\\__init__.py', reloading
[2025-04-29 19:21:35,458] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\_pytest\\cacheprovider.py', reloading
[2025-04-29 19:21:35,466] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\_pytest\\capture.py', reloading
[2025-04-29 19:21:35,467] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\_pytest\\config\\__init__.py', reloading
[2025-04-29 19:21:35,474] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\_pytest\\config\\exceptions.py', reloading
[2025-04-29 19:21:35,480] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\_pytest\\config\\argparsing.py', reloading
[2025-04-29 19:21:35,485] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\_pytest\\fixtures.py', reloading
[2025-04-29 19:21:35,492] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\_pytest\\freeze_support.py', reloading
[2025-04-29 19:21:35,492] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\_pytest\\legacypath.py', reloading
[2025-04-29 19:21:35,502] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\_pytest\\logging.py', reloading
[2025-04-29 19:21:35,508] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\_pytest\\main.py', reloading
[2025-04-29 19:21:35,512] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\_pytest\\mark\\__init__.py', reloading
[2025-04-29 19:21:35,516] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\_pytest\\mark\\structures.py', reloading
[2025-04-29 19:21:35,517] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\_pytest\\monkeypatch.py', reloading
[2025-04-29 19:21:35,522] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\_pytest\\nodes.py', reloading
[2025-04-29 19:21:35,526] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\_pytest\\outcomes.py', reloading
[2025-04-29 19:21:35,530] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\_pytest\\pytester.py', reloading
[2025-04-29 19:21:35,534] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\_pytest\\python.py', reloading
[2025-04-29 19:21:35,547] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\_pytest\\python_api.py', reloading
[2025-04-29 19:21:35,550] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\_pytest\\recwarn.py', reloading
[2025-04-29 19:21:35,554] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\_pytest\\reports.py', reloading
[2025-04-29 19:21:35,555] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\_pytest\\runner.py', reloading
[2025-04-29 19:21:35,560] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\_pytest\\stash.py', reloading
[2025-04-29 19:21:35,565] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\_pytest\\terminal.py', reloading
[2025-04-29 19:21:35,574] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\_pytest\\tmpdir.py', reloading
[2025-04-29 19:21:35,576] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\_pytest\\warning_types.py', reloading
[2025-04-29 19:21:35,576] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\pytoolconfig\\__init__.py', reloading
[2025-04-29 19:21:35,583] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\pytoolconfig\\fields.py', reloading
[2025-04-29 19:21:35,586] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\pytoolconfig\\pytoolconfig.py', reloading
[2025-04-29 19:21:35,589] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\pytoolconfig\\types.py', reloading
[2025-04-29 19:21:35,592] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\pyviz_comms\\__init__.py', reloading
[2025-04-29 19:21:35,595] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\pywt\\__init__.py', reloading
[2025-04-29 19:21:35,600] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\pywt\\_functions.py', reloading
[2025-04-29 19:21:35,608] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\pywt\\_multilevel.py', reloading
[2025-04-29 19:21:35,616] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\pywt\\_multidim.py', reloading
[2025-04-29 19:21:35,621] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\pywt\\_thresholding.py', reloading
[2025-04-29 19:21:35,625] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\pywt\\_wavelet_packets.py', reloading
[2025-04-29 19:21:35,632] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\pywt\\_dwt.py', reloading
[2025-04-29 19:21:35,633] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\pywt\\_swt.py', reloading
[2025-04-29 19:21:35,639] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\pywt\\_cwt.py', reloading
[2025-04-29 19:21:35,649] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\pywt\\_mra.py', reloading
[2025-04-29 19:21:35,654] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\qdarkstyle\\__init__.py', reloading
[2025-04-29 19:21:35,656] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\qstylizer\\__init__.py', reloading
[2025-04-29 19:21:35,658] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\qtawesome\\__init__.py', reloading
[2025-04-29 19:21:35,658] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\qtconsole\\__init__.py', reloading
[2025-04-29 19:21:35,661] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\qtpy\\__init__.py', reloading
[2025-04-29 19:21:35,669] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\queuelib\\__init__.py', reloading
[2025-04-29 19:21:35,672] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\referencing\\__init__.py', reloading
[2025-04-29 19:21:35,675] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\referencing\\_core.py', reloading
[2025-04-29 19:21:35,675] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\repo_cli\\__init__.py', reloading
[2025-04-29 19:21:35,681] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\requests_toolbelt\\__init__.py', reloading
[2025-04-29 19:21:35,684] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\requests_toolbelt\\adapters\\__init__.py', reloading
[2025-04-29 19:21:35,690] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\requests_toolbelt\\adapters\\ssl.py', reloading
[2025-04-29 19:21:35,692] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\requests_toolbelt\\adapters\\source.py', reloading
[2025-04-29 19:21:35,692] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\requests_toolbelt\\auth\\guess.py', reloading
[2025-04-29 19:21:35,696] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\requests_toolbelt\\multipart\\__init__.py', reloading
[2025-04-29 19:21:35,698] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\requests_toolbelt\\multipart\\encoder.py', reloading
[2025-04-29 19:21:35,700] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\requests_toolbelt\\multipart\\decoder.py', reloading
[2025-04-29 19:21:35,704] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\requests_toolbelt\\streaming_iterator.py', reloading
[2025-04-29 19:21:35,706] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\requests_toolbelt\\utils\\user_agent.py', reloading
[2025-04-29 19:21:35,709] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\rich\\__init__.py', reloading
[2025-04-29 19:21:35,709] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\rope\\__init__.py', reloading
[2025-04-29 19:21:35,717] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\rtree\\__init__.py', reloading
[2025-04-29 19:21:35,717] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\ruamel_yaml\\__init__.py', reloading
[2025-04-29 19:21:35,717] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\s3fs\\__init__.py', reloading
[2025-04-29 19:21:35,722] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\cluster\\__init__.py', reloading
[2025-04-29 19:21:35,727] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\conftest.py', reloading
[2025-04-29 19:21:35,727] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\constants\\__init__.py', reloading
[2025-04-29 19:21:35,733] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\constants\\_codata.py', reloading
[2025-04-29 19:21:35,747] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\constants\\_constants.py', reloading
[2025-04-29 19:21:35,749] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\datasets\\__init__.py', reloading
[2025-04-29 19:21:35,755] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\datasets\\_fetchers.py', reloading
[2025-04-29 19:21:35,758] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\datasets\\_download_all.py', reloading
[2025-04-29 19:21:35,758] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\datasets\\_utils.py', reloading
[2025-04-29 19:21:35,763] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\fft\\__init__.py', reloading
[2025-04-29 19:21:35,767] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\fft\\_basic.py', reloading
[2025-04-29 19:21:35,767] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\fft\\_realtransforms.py', reloading
[2025-04-29 19:21:35,773] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\fft\\_fftlog.py', reloading
[2025-04-29 19:21:35,775] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\fft\\_fftlog_backend.py', reloading
[2025-04-29 19:21:35,782] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\fft\\_helper.py', reloading
[2025-04-29 19:21:35,783] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\fft\\_backend.py', reloading
[2025-04-29 19:21:35,783] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\fft\\_pocketfft\\helper.py', reloading
[2025-04-29 19:21:35,791] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\integrate\\__init__.py', reloading
[2025-04-29 19:21:35,797] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\integrate\\_quadrature.py', reloading
[2025-04-29 19:21:35,805] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\integrate\\_odepack_py.py', reloading
[2025-04-29 19:21:35,809] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\integrate\\_quadpack_py.py', reloading
[2025-04-29 19:21:35,810] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\integrate\\_ode.py', reloading
[2025-04-29 19:21:35,817] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\interpolate\\__init__.py', reloading
[2025-04-29 19:21:35,825] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\interpolate\\_interpolate.py', reloading
[2025-04-29 19:21:35,833] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\interpolate\\_fitpack_py.py', reloading
[2025-04-29 19:21:35,836] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\interpolate\\_fitpack2.py', reloading
[2025-04-29 19:21:35,842] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\interpolate\\_rbfinterp.py', reloading
[2025-04-29 19:21:35,849] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\interpolate\\_polyint.py', reloading
[2025-04-29 19:21:35,850] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\interpolate\\_cubic.py', reloading
[2025-04-29 19:21:35,856] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\interpolate\\_ndgriddata.py', reloading
[2025-04-29 19:21:35,858] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\interpolate\\_bsplines.py', reloading
[2025-04-29 19:21:35,867] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\interpolate\\_pade.py', reloading
[2025-04-29 19:21:35,867] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\interpolate\\_rgi.py', reloading
[2025-04-29 19:21:35,875] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\io\\__init__.py', reloading
[2025-04-29 19:21:35,883] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\misc\\__init__.py', reloading
[2025-04-29 19:21:35,887] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\misc\\_common.py', reloading
[2025-04-29 19:21:35,891] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\ndimage\\__init__.py', reloading
[2025-04-29 19:21:35,895] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\ndimage\\_filters.py', reloading
[2025-04-29 19:21:35,900] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\ndimage\\_fourier.py', reloading
[2025-04-29 19:21:35,903] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\ndimage\\_interpolation.py', reloading
[2025-04-29 19:21:35,905] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\ndimage\\_measurements.py', reloading
[2025-04-29 19:21:35,908] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\ndimage\\_morphology.py', reloading
[2025-04-29 19:21:35,917] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\odr\\__init__.py', reloading
[2025-04-29 19:21:35,917] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\odr\\_odrpack.py', reloading
[2025-04-29 19:21:35,924] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\odr\\_models.py', reloading
[2025-04-29 19:21:35,927] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\signal\\__init__.py', reloading
[2025-04-29 19:21:35,933] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\signal\\_waveforms.py', reloading
[2025-04-29 19:21:35,937] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\signal\\_bsplines.py', reloading
[2025-04-29 19:21:35,942] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\signal\\_filter_design.py', reloading
[2025-04-29 19:21:35,950] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\signal\\_fir_filter_design.py', reloading
[2025-04-29 19:21:35,950] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\signal\\_ltisys.py', reloading
[2025-04-29 19:21:35,959] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\signal\\_lti_conversion.py', reloading
[2025-04-29 19:21:35,965] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\signal\\_signaltools.py', reloading
[2025-04-29 19:21:35,974] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\signal\\_spectral_py.py', reloading
[2025-04-29 19:21:35,978] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\signal\\_short_time_fft.py', reloading
[2025-04-29 19:21:35,983] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\signal\\_wavelets.py', reloading
[2025-04-29 19:21:35,990] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\signal\\_peak_finding.py', reloading
[2025-04-29 19:21:36,001] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\signal\\_czt.py', reloading
[2025-04-29 19:21:36,011] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\spatial\\__init__.py', reloading
[2025-04-29 19:21:36,016] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\spatial\\_kdtree.py', reloading
[2025-04-29 19:21:36,034] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\spatial\\_plotutils.py', reloading
[2025-04-29 19:21:36,041] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\special\\_sf_error.py', reloading
[2025-04-29 19:21:36,045] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\special\\_spfun_stats.py', reloading
[2025-04-29 19:21:36,049] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\special\\_ellip_harm.py', reloading
[2025-04-29 19:21:36,051] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\special\\_lambertw.py', reloading
[2025-04-29 19:21:36,059] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\special\\_spherical_bessel.py', reloading
[2025-04-29 19:21:36,059] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\version.py', reloading
[2025-04-29 19:21:36,068] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scipy\\__config__.py', reloading
[2025-04-29 19:21:36,075] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scrapy\\__init__.py', reloading
[2025-04-29 19:21:36,092] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scrapy\\http\\__init__.py', reloading
[2025-04-29 19:21:36,098] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scrapy\\http\\request\\form.py', reloading
[2025-04-29 19:21:36,109] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scrapy\\http\\request\\__init__.py', reloading
[2025-04-29 19:21:36,118] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scrapy\\item.py', reloading
[2025-04-29 19:21:36,126] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scrapy\\selector\\__init__.py', reloading
[2025-04-29 19:21:36,126] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scrapy\\selector\\unified.py', reloading
[2025-04-29 19:21:36,134] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\scrapy\\spiders\\__init__.py', reloading
[2025-04-29 19:21:36,134] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\semver\\__init__.py', reloading
[2025-04-29 19:21:36,142] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\semver\\_deprecated.py', reloading
[2025-04-29 19:21:36,142] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\semver\\version.py', reloading
[2025-04-29 19:21:36,142] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\semver\\__about__.py', reloading
[2025-04-29 19:21:36,150] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\service_identity\\__init__.py', reloading
[2025-04-29 19:21:36,153] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\service_identity\\exceptions.py', reloading
[2025-04-29 19:21:36,153] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\sipbuild\\__init__.py', reloading
[2025-04-29 19:21:36,153] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\slugify\\__init__.py', reloading
[2025-04-29 19:21:36,158] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\smart_open\\__init__.py', reloading
[2025-04-29 19:21:36,192] INFO in _internal:  * Restarting with watchdog (windowsapi)
[2025-04-29 19:21:36,432] WARNING in _internal:  * Debugger is active!
[2025-04-29 19:21:36,441] INFO in _internal:  * Debugger PIN: 131-854-612
[2025-04-29 19:21:36,458] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\streamlit\\echo.py', reloading
[2025-04-29 19:21:36,467] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\streamlit\\commands\\logo.py', reloading
[2025-04-29 19:21:36,469] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\streamlit\\commands\\navigation.py', reloading
[2025-04-29 19:21:36,475] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\streamlit\\navigation\\page.py', reloading
[2025-04-29 19:21:36,475] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\streamlit\\elements\\spinner.py', reloading
[2025-04-29 19:21:36,475] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\streamlit\\commands\\page_config.py', reloading
[2025-04-29 19:21:36,483] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\streamlit\\commands\\execution_control.py', reloading
[2025-04-29 19:21:36,490] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\tables\\__init__.py', reloading
[2025-04-29 19:21:36,492] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\tables\\atom.py', reloading
[2025-04-29 19:21:36,502] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\tables\\description.py', reloading
[2025-04-29 19:21:36,509] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\tables\\exceptions.py', reloading
[2025-04-29 19:21:36,512] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\tables\\misc\\enum.py', reloading
[2025-04-29 19:21:36,515] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\tables\\flavor.py', reloading
[2025-04-29 19:21:36,517] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\tables\\filters.py', reloading
[2025-04-29 19:21:36,522] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\tables\\file.py', reloading
[2025-04-29 19:21:36,532] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\tables\\node.py', reloading
[2025-04-29 19:21:36,540] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\tables\\group.py', reloading
[2025-04-29 19:21:36,542] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\tables\\leaf.py', reloading
[2025-04-29 19:21:36,551] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\tables\\table.py', reloading
[2025-04-29 19:21:36,558] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\tables\\array.py', reloading
[2025-04-29 19:21:36,562] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\tables\\carray.py', reloading
[2025-04-29 19:21:36,565] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\tables\\earray.py', reloading
[2025-04-29 19:21:36,567] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\tables\\vlarray.py', reloading
[2025-04-29 19:21:36,569] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\tables\\unimplemented.py', reloading
[2025-04-29 19:21:36,575] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\tables\\expression.py', reloading
[2025-04-29 19:21:36,575] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\tables\\tests\\__init__.py', reloading
[2025-04-29 19:21:36,582] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\tables\\tests\\common.py', reloading
[2025-04-29 19:21:36,584] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\tables\\tests\\test_suite.py', reloading
[2025-04-29 19:21:36,584] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\tblib\\__init__.py', reloading
[2025-04-29 19:21:36,593] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\tenacity\\__init__.py', reloading
[2025-04-29 19:21:36,600] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\tenacity\\retry.py', reloading
[2025-04-29 19:21:36,601] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\tenacity\\nap.py', reloading
[2025-04-29 19:21:36,601] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\tenacity\\stop.py', reloading
[2025-04-29 19:21:36,606] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\tenacity\\wait.py', reloading
[2025-04-29 19:21:36,609] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\tenacity\\before.py', reloading
[2025-04-29 19:21:36,610] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\tenacity\\after.py', reloading
[2025-04-29 19:21:36,612] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\tenacity\\before_sleep.py', reloading
[2025-04-29 19:21:36,614] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\tenacity\\_asyncio.py', reloading
[2025-04-29 19:21:36,615] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\terminado\\__init__.py', reloading
[2025-04-29 19:21:36,617] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\textdistance\\__init__.py', reloading
[2025-04-29 19:21:36,617] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\text_unidecode\\__init__.py', reloading
[2025-04-29 19:21:36,617] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\three_merge\\__init__.py', reloading
[2025-04-29 19:21:36,623] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\tifffile\\__init__.py', reloading
[2025-04-29 19:21:36,626] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\tifffile\\tifffile.py', reloading
[2025-04-29 19:21:36,674] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\tinycss2\\__init__.py', reloading
[2025-04-29 19:21:36,674] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\tldextract\\__init__.py', reloading
[2025-04-29 19:21:36,674] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\tldextract\\tldextract.py', reloading
[2025-04-29 19:21:36,683] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\tlz\\__init__.py', reloading
[2025-04-29 19:21:36,690] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\tomli\\__init__.py', reloading
[2025-04-29 19:21:36,692] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\tomli\\_parser.py', reloading
[2025-04-29 19:21:36,695] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\tomlkit\\__init__.py', reloading
[2025-04-29 19:21:36,700] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\tomlkit\\api.py', reloading
[2025-04-29 19:21:36,700] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\tomlkit\\items.py', reloading
[2025-04-29 19:21:36,709] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\toolz\\__init__.py', reloading
[2025-04-29 19:21:36,709] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\tornado\\__init__.py', reloading
[2025-04-29 19:21:36,709] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\traitlets\\__init__.py', reloading
[2025-04-29 19:21:36,716] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\traitlets\\traitlets.py', reloading
[2025-04-29 19:21:36,726] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\traitlets\\_version.py', reloading
[2025-04-29 19:21:36,732] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\traitlets\\utils\\bunch.py', reloading
[2025-04-29 19:21:36,734] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\traitlets\\utils\\decorators.py', reloading
[2025-04-29 19:21:36,734] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\traitlets\\utils\\importstring.py', reloading
[2025-04-29 19:21:36,734] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\truststore\\__init__.py', reloading
[2025-04-29 19:21:36,742] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\truststore\\_api.py', reloading
[2025-04-29 19:21:36,742] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\twisted\\__init__.py', reloading
[2025-04-29 19:21:36,749] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\tzdata\\__init__.py', reloading
[2025-04-29 19:21:36,749] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\uc_micro\\__init__.py', reloading
[2025-04-29 19:21:36,752] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\unidecode\\__init__.py', reloading
[2025-04-29 19:21:36,754] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\urllib3\\__init__.py', reloading
[2025-04-29 19:21:36,758] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\urllib3\\_collections.py', reloading
[2025-04-29 19:21:36,758] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\urllib3\\connectionpool.py', reloading
[2025-04-29 19:21:36,765] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\urllib3\\filepost.py', reloading
[2025-04-29 19:21:36,767] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\urllib3\\poolmanager.py', reloading
[2025-04-29 19:21:36,767] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\urllib3\\response.py', reloading
[2025-04-29 19:21:36,775] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\urllib3\\util\\request.py', reloading
[2025-04-29 19:21:36,778] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\urllib3\\util\\retry.py', reloading
[2025-04-29 19:21:36,781] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\urllib3\\util\\timeout.py', reloading
[2025-04-29 19:21:36,783] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\w3lib\\__init__.py', reloading
[2025-04-29 19:21:36,786] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\watchdog\\__init__.py', reloading
[2025-04-29 19:21:36,788] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\wcwidth\\__init__.py', reloading
[2025-04-29 19:21:36,790] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\wcwidth\\wcwidth.py', reloading
[2025-04-29 19:21:36,793] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\wcwidth\\unicode_versions.py', reloading
[2025-04-29 19:21:36,798] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\webencodings\\__init__.py', reloading
[2025-04-29 19:21:36,800] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\websocket\\__init__.py', reloading
[2025-04-29 19:21:36,800] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\websocket\\_abnf.py', reloading
[2025-04-29 19:21:36,806] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\websocket\\_core.py', reloading
[2025-04-29 19:21:36,810] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\websocket\\_exceptions.py', reloading
[2025-04-29 19:21:36,810] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\websocket\\_logging.py', reloading
[2025-04-29 19:21:36,814] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\websocket\\_socket.py', reloading
[2025-04-29 19:21:36,817] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\websocket\\_app.py', reloading
[2025-04-29 19:21:36,817] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\werkzeug\\__init__.py', reloading
[2025-04-29 19:21:36,823] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\werkzeug\\serving.py', reloading
[2025-04-29 19:21:36,825] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\werkzeug\\test.py', reloading
[2025-04-29 19:21:36,834] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\werkzeug\\wrappers\\response.py', reloading
[2025-04-29 19:21:36,839] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\wheel\\__init__.py', reloading
[2025-04-29 19:21:36,839] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\widgetsnbextension\\__init__.py', reloading
[2025-04-29 19:21:36,842] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\win32ctypes\\__init__.py', reloading
[2025-04-29 19:21:36,842] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\winpty\\__init__.py', reloading
[2025-04-29 19:21:36,846] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\wrapt\\__init__.py', reloading
[2025-04-29 19:21:36,848] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\xarray\\__init__.py', reloading
[2025-04-29 19:21:36,857] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\xarray\\backends\\api.py', reloading
[2025-04-29 19:21:36,867] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\xarray\\backends\\zarr.py', reloading
[2025-04-29 19:21:36,867] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\xarray\\coding\\cftime_offsets.py', reloading
[2025-04-29 19:21:36,876] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\xarray\\coding\\cftimeindex.py', reloading
[2025-04-29 19:21:36,881] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\xarray\\coding\\frequencies.py', reloading
[2025-04-29 19:21:36,883] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\xarray\\conventions.py', reloading
[2025-04-29 19:21:36,886] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\xarray\\coding\\variables.py', reloading
[2025-04-29 19:21:36,893] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\xarray\\core\\alignment.py', reloading
[2025-04-29 19:21:36,898] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\xarray\\core\\combine.py', reloading
[2025-04-29 19:21:36,900] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\xarray\\core\\common.py', reloading
[2025-04-29 19:21:36,907] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\xarray\\core\\computation.py', reloading
[2025-04-29 19:21:36,912] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\xarray\\core\\concat.py', reloading
[2025-04-29 19:21:36,917] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\xarray\\core\\dataarray.py', reloading
[2025-04-29 19:21:36,932] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\xarray\\core\\dataset.py', reloading
[2025-04-29 19:21:36,942] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\xarray\\core\\extensions.py', reloading
[2025-04-29 19:21:36,947] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\xarray\\core\\merge.py', reloading
[2025-04-29 19:21:36,951] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\xarray\\core\\options.py', reloading
[2025-04-29 19:21:36,955] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\xarray\\core\\parallel.py', reloading
[2025-04-29 19:21:36,959] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\xarray\\core\\variable.py', reloading
[2025-04-29 19:21:36,968] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\xarray\\util\\print_versions.py', reloading
[2025-04-29 19:21:36,968] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\xlwings\\__init__.py', reloading
[2025-04-29 19:21:36,977] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\xlwings\\main.py', reloading
[2025-04-29 19:21:36,983] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\xlwings\\utils.py', reloading
[2025-04-29 19:21:36,992] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\xyzservices\\__init__.py', reloading
[2025-04-29 19:21:36,992] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\yapf\\__init__.py', reloading
[2025-04-29 19:21:36,999] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\yapftests\\__init__.py', reloading
[2025-04-29 19:21:36,999] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\yarl\\__init__.py', reloading
[2025-04-29 19:21:37,003] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\yarl\\_url.py', reloading
[2025-04-29 19:21:37,008] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\zict\\__init__.py', reloading
[2025-04-29 19:21:37,010] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\zict\\async_buffer.py', reloading
[2025-04-29 19:21:37,017] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\zict\\buffer.py', reloading
[2025-04-29 19:21:37,017] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\zict\\cache.py', reloading
[2025-04-29 19:21:37,022] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\zict\\file.py', reloading
[2025-04-29 19:21:37,024] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\zict\\func.py', reloading
[2025-04-29 19:21:37,028] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\zict\\lmdb.py', reloading
[2025-04-29 19:21:37,032] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\zict\\lru.py', reloading
[2025-04-29 19:21:37,035] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\zict\\sieve.py', reloading
[2025-04-29 19:21:37,035] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\zict\\utils.py', reloading
[2025-04-29 19:21:37,039] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\zict\\zip.py', reloading
[2025-04-29 19:21:37,042] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\zipp\\__init__.py', reloading
[2025-04-29 19:21:37,050] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\zmq\\constants.py', reloading
[2025-04-29 19:21:37,058] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\zmq\\error.py', reloading
[2025-04-29 19:21:37,062] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\zmq\\sugar\\context.py', reloading
[2025-04-29 19:21:37,065] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\zmq\\sugar\\frame.py', reloading
[2025-04-29 19:21:37,067] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\zmq\\sugar\\poll.py', reloading
[2025-04-29 19:21:37,070] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\zmq\\sugar\\socket.py', reloading
[2025-04-29 19:21:37,073] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\zmq\\sugar\\tracker.py', reloading
[2025-04-29 19:21:37,073] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\zmq\\sugar\\version.py', reloading
[2025-04-29 19:21:37,082] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\zope\\__init__.py', reloading
[2025-04-29 19:21:37,090] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\pythonwin\\pywin\\__init__.py', reloading
[2025-04-29 19:21:37,483] INFO in _internal:  * Restarting with watchdog (windowsapi)
[2025-04-29 19:21:37,697] WARNING in _internal:  * Debugger is active!
[2025-04-29 19:21:37,698] INFO in _internal:  * Debugger PIN: 131-854-612
[2025-04-29 19:25:36,810] INFO in _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
[2025-04-29 19:25:36,810] INFO in _internal: [33mPress CTRL+C to quit[0m
[2025-04-29 19:25:36,819] INFO in _internal:  * Restarting with watchdog (windowsapi)
[2025-04-29 19:25:38,019] WARNING in _internal:  * Debugger is active!
[2025-04-29 19:25:38,019] INFO in _internal:  * Debugger PIN: 122-965-387
[2025-04-29 19:29:08,158] INFO in _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
[2025-04-29 19:29:08,158] INFO in _internal: [33mPress CTRL+C to quit[0m
[2025-04-29 19:29:08,168] INFO in _internal:  * Restarting with watchdog (windowsapi)
[2025-04-29 19:29:08,395] WARNING in _internal:  * Debugger is active!
[2025-04-29 19:29:08,395] INFO in _internal:  * Debugger PIN: 131-854-612
[2025-04-29 19:34:29,816] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\logging\\__init__.py', reloading
[2025-04-29 19:34:29,936] INFO in _internal:  * Restarting with watchdog (windowsapi)
[2025-04-29 19:34:30,223] WARNING in _internal:  * Debugger is active!
[2025-04-29 19:34:30,223] INFO in _internal:  * Debugger PIN: 131-854-612
[2025-04-29 19:41:11,636] INFO in _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
[2025-04-29 19:41:11,636] INFO in _internal: [33mPress CTRL+C to quit[0m
[2025-04-29 19:41:11,643] INFO in _internal:  * Restarting with watchdog (windowsapi)
[2025-04-29 19:41:11,860] WARNING in _internal:  * Debugger is active!
[2025-04-29 19:41:11,860] INFO in _internal:  * Debugger PIN: 131-854-612
[2025-04-29 19:42:11,108] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\datetime.py', reloading
[2025-04-29 19:42:11,138] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\_pydatetime.py', reloading
[2025-04-29 19:42:11,988] INFO in _internal:  * Restarting with watchdog (windowsapi)
[2025-04-29 19:42:12,235] WARNING in _internal:  * Debugger is active!
[2025-04-29 19:42:12,235] INFO in _internal:  * Debugger PIN: 131-854-612
[2025-04-29 20:20:43,395] INFO in _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
[2025-04-29 20:20:43,395] INFO in _internal: [33mPress CTRL+C to quit[0m
[2025-04-29 20:20:43,415] INFO in _internal:  * Restarting with watchdog (windowsapi)
[2025-04-29 20:20:43,641] WARNING in _internal:  * Debugger is active!
[2025-04-29 20:20:43,641] INFO in _internal:  * Debugger PIN: 131-854-612
[2025-04-29 20:20:53,517] INFO in _internal: 127.0.0.1 - - [29/Apr/2025 20:20:53] "GET / HTTP/1.1" 200 -
[2025-04-29 20:20:53,631] INFO in _internal: 127.0.0.1 - - [29/Apr/2025 20:20:53] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -
[2025-04-29 20:20:59,774] INFO in app: Requst data: {'area': '30', 'rooms': '', 'total_floors': '', 'floor': ''}
[2025-04-29 20:20:59,775] INFO in _internal: 127.0.0.1 - - [29/Apr/2025 20:20:59] "POST /api/numbers HTTP/1.1" 200 -
[2025-04-29 20:21:16,627] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\flask\\typing.py', reloading
[2025-04-29 20:21:16,641] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\werkzeug\\sansio\\response.py', reloading
[2025-04-29 20:21:16,659] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\werkzeug\\datastructures\\__init__.py', reloading
[2025-04-29 20:21:16,674] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\werkzeug\\utils.py', reloading
[2025-04-29 20:21:16,691] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\flask\\config.py', reloading
[2025-04-29 20:21:16,701] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\flask\\globals.py', reloading
[2025-04-29 20:21:16,701] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\flask\\wrappers.py', reloading
[2025-04-29 20:21:16,708] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\werkzeug\\wrappers\\__init__.py', reloading
[2025-04-29 20:21:16,709] INFO in _internal:  * Detected change in 'D:\\Anaconda\\Lib\\site-packages\\werkzeug\\wrappers\\request.py', reloading
[2025-04-29 20:21:16,749] INFO in _internal:  * Restarting with watchdog (windowsapi)
[2025-04-29 20:21:16,981] WARNING in _internal:  * Debugger is active!
[2025-04-29 20:21:16,990] INFO in _internal:  * Debugger PIN: 131-854-612
